### PR TITLE
Adding id parameter to Action object

### PR DESCRIPTION
**Pull request by NetaMolcho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
 		"package": "vsce package ."
 	},
 	"dependencies": {
+		"@sap/consume-services-types": "^0.3.6",
```
Comment (idoprz) [inline package.json:45]: The toolkit should not be dependant on the consume services. It is lower level extension.
You should either define the IService in the toolkit, or use some more "abstract type" (preferable not any)

File: src/actions/interfaces.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import * as vscode from "vscode";
+import { IService } from "@sap/consume-services-types";
```
Comment (idoprz) [inline src/actions/interfaces.ts:2]: remove this dependency...

File: src/actions/interfaces.ts [hunk 3]
```diff
@@ -17,10 +18,11 @@ export enum ActionJsonKey {
 
 export type CommandActionParams = any [];
 export type ExecuteActionParams = any [];
-export type SnippetActionParams = any;
+export type SnippetActionParams = any | {data: any, service: IService} | {data: any};
```
Comment (idoprz) [inline src/actions/interfaces.ts:21]: it can be service: any
or better to define something in here that will cover the service type

**Review by idoprz — COMMENTED** [review]
see my comments in line. otherwise it's fine by me, but Tomer should approve. not me.

**Review by idoprz — DISMISSED** [review]
👍

**Review by tomer-epstein — CHANGES_REQUESTED** [review]
increment version 0.0.4--> https://github.com/NetaMolcho/app-studio-toolkit/blob/idAction/package.json#L5
increment version 0.0.4 (too) --> https://github.com/NetaMolcho/app-studio-toolkit/blob/idAction/types/package.json#L4

**Review by tomer-epstein — APPROVED** [review]

**Pull request merged**